packages/hooks/src/useP2PAdvertList.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useFetch, useInvalidateQuery } from '@deriv/api';
+import useExchangeRate from './useExchangeRate';
+
+/**
+ * This custom hook returns available adverts for use with 'p2p_order_create' by calling 'p2p_advert_list' endpoint
```

Comment:
`non-blocking:` just my personal preference but could we add something like this just so we can see what it'll return 

```suggestion
 * This custom hook returns available adverts for use with 'p2p_order_create' by calling 'p2p_advert_list' endpoint
 * @return {object} p2p_advert_list` 
```

Reply:
Sure will add this :pray: